tests/paz/pipelines/human_pose_3d_tests.py

```diff
@@ -1,26 +1,50 @@
 import pytest
-from pipeline import SimpleBaselines
+from paz.pipelines.keypoints import SimpleBaselines
 import numpy as np
+import os
 from paz.backend.image import load_image
 from paz.backend.camera import Camera
-from linear_model import Simple_Baseline
-from keypoints_processors import SolveTranslation3D
+from paz.models.keypoint.simplebaselines import SimpleBaseline
 from scipy.optimize import least_squares
+from tensorflow.keras.utils import get_file
+from paz.backend.keypoints import filter_keypoints3D
+from paz.backend.keypoints import initialize_translation, solve_least_squares,\
+    get_bones_length, compute_reprojection_error,\
+    compute_optimized_pose3D
 
 
 h36m_to_coco_joints2D = [4, 12, 14, 16, 11, 13, 15, 2, 1, 0, 5, 7, 9, 6, 8, 10]
 args_to_joints3D = [0, 1, 2, 3, 6, 7, 8, 12, 13, 15, 17, 18, 19, 25, 26, 27]
 args_to_mean = {1: [5, 6], 4: [11, 12], 2: [1, 4]}
```

Comment:
this should not be used here. since they will be hardcoded we would not need this anymore. So please remove.